lib/jenkins_pipeline_builder/pull_request_generator.rb

```diff
@@ -30,16 +30,16 @@ def initialize(defaults = {})
     end
 
     def convert!(job_collection, pr)
```

Comment:
Since the `#open_prs` needs to return more than just the PR number now, I decided to change the other signatures that accept `pr` to remain consistent though I'm not sure if there is anything depending on this outside of the gem. If so, I'm happy to revert back to just the number.

Reply:
As I type this comment I think I want to change the signature back 😄  though if truly private and not depended on anywhere else this could make for a nice start for 2.0